HIP-xxxx.md

```diff
@@ -0,0 +1,41 @@
+# HIP-xxxx : Name data standard for alternative namespace resolution
+
+```
+Number:  HIP-xxxx
+Title:   Name data standard for alternative namespace resolution
+Type:    Informational
+Status:  Draft
+Authors: Mike Carson <https://impervious.com/>
+Created: 2021-02-08
+```
+
+## Abstract
+
+This HIP describes how to set the data for a Handshake name so that it resolves via an alternative namespace, for example via a sidechain.
+
+## Motivation
+
+There may be various reasons to resolve a name on an alternative namespace other than a domain name or an ip address.  One example is for decentralized subdomains on Ethereum.  Software could recognize the data on a name in a way to resolve it with Ethereum.
+
+## Name data standard
+
+We propose setting the NS records for the name with an underscore suffix which will resolve to an alternative protocol.  The prefix will be used as data for the protocol. For example:
+
+```
+{
+  "records": [{
+    "type": "NS",
+    "ns": "0x36fc69f0983E536D1787cC83f481581f22CCA2A1._eth."
```

Comment:
This is probably going to change as we work on the actual implementation. Let's hold off on merging this HIP until something works ;-)

Reply:
this didn't end up changing with our implementation, so I assume we can merge this now